conftest.py

```diff
@@ -68,6 +71,11 @@ def pytest_configure(config):
 
     config.custom_config = custom_config.CustomConfig(config.getoption('--config'))
 
+    if any([x for x in config.args if 'gsuite' in x]):
```

Comment:
We should be able to drop the [] brackets since the any will consume the generator.